test/corpus/type-fields.txt

```diff
@@ -125,52 +124,40 @@ struct A {
             (function_pointer_parameter
               type: (function_pointer_type
                 (function_pointer_parameter
-                  (parameter_modifier)
                   type: (predefined_type))
-                (function_pointer_parameter
-                  type: (predefined_type))))
-            (function_pointer_parameter
-              type: (function_pointer_type
-                (function_pointer_parameter
-                  (parameter_modifier)
-                  type: (predefined_type))
-                (function_pointer_parameter
-                  type: (ref_type
-                    type: (predefined_type))))))
+                returns: (predefined_type)))
+            returns: (function_pointer_type
+              (function_pointer_parameter
+                type: (predefined_type))
+              returns: (ref_type
+                type: (predefined_type))))
           (variable_declarator
             name: (identifier))))
       (comment)
       (field_declaration
         (variable_declaration
           type: (function_pointer_type
-            (function_pointer_calling_convention)
+            (calling_convention)
             (function_pointer_parameter
               type: (predefined_type))
-            (function_pointer_parameter
-              type: (predefined_type)))
+            returns: (predefined_type))
           (variable_declarator
             name: (identifier))))
       (field_declaration
         (variable_declaration
           type: (function_pointer_type
             (function_pointer_parameter
               type: (function_pointer_type
-                (function_pointer_calling_convention
-                  (function_pointer_unmanaged_calling_convention_list
-                    (function_pointer_unmanaged_calling_convention
-                      (identifier))
-                    (function_pointer_unmanaged_calling_convention
-                      (identifier))))
+                (calling_convention
```

Comment:
The function pointer calling convention has the old term in the grammar but it's definitely an less used construct so if this shared calling_convention results in an easier/shorter grammar I'm 👍🏻